x/concentrated-liquidity/incentives_test.go

```diff
@@ -2678,66 +2580,27 @@ func (s *KeeperTestSuite) TestCollectIncentives() {
 			addedUptimeGrowthInside:  uptimeHelper.hundredTokensMultiDenom,
 			addedUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
 			positionParams: positionParameters{
-				owner:          ownerWithValidPosition,
-				lowerTick:      0,
-				upperTick:      2,
-				liquidity:      DefaultLiquidityAmt,
-				joinTime:       defaultJoinTime,
-				freezeDuration: oneDayFreeze,
-				positionId:     DefaultPositionId,
-				collectTime:    defaultJoinTime.Add(100),
-			},
-			numPositions: 1,
-
+				owner:       ownerWithValidPosition,
+				lowerTick:   0,
+				upperTick:   2,
+				liquidity:   DefaultLiquidityAmt,
+				joinTime:    defaultJoinTime,
+				positionId:  DefaultPositionId,
+				collectTime: defaultJoinTime.Add(100),
+			},
+			timeInPosition: oneDay,
 			// Since there is no other existing liquidity, we expect all of the growth inside to accrue to be claimed for the
 			// uptimes the position qualifies for.
-			expectedIncentivesClaimed: expectedIncentivesFromUptimeGrowth(uptimeHelper.hundredTokensMultiDenom, DefaultLiquidityAmt, oneDayFreeze, defaultMultiplier),
```

Comment:
Why is this one removed?

Reply:
So this is the only test case that tests multiple positions that have the same range. Previously, this was necessary to test because these positions would be combined as one under the hood when you claimed. We now treat every position independently (it could be one position or 100 position in the same range, a single position ID is needed to call claim). Therefore, this test felt obsolete. 

Please let me know what you think.

Reply:
I think it might still be a useful test case depicting the exact behavior that you described - if you have multiple positions within a range, you claim only for one within that range by id

Reply:
Agreed, added it back here! 

https://github.com/osmosis-labs/osmosis/pull/4724/commits/c44b0fe018e9f662eb0d0bc97472db0ac5a1e91b